erizo_controller/erizoClient/src/webrtc-stacks/BaseStack.js

```diff
@@ -14,7 +14,11 @@ import FunctionQueue from '../utils/FunctionQueue';
 const BaseStack = (specInput) => {
   const that = {};
   const specBase = specInput;
-  const negotiationQueue = new FunctionQueue();
+  const negotiationQueue = new FunctionQueue(30000, () => {
```

Comment:
Consider using a constant somewhere instead of using the `30000` value so it's easier to see what it's for

Reply:
good catch!